components/CompanyRegister.tsx

```diff
@@ -49,80 +79,65 @@ const CompanyRegister = () => {
       const response = await fetch('api/companies', options);
       if (response.ok) {
         const responseData = await response.json();
-
         if (isSuccessResponse(responseData)) {
           toast.success(`${responseData.item.name} Registered Successfully`);
           router.push('/');
+        } else {
+          toast.error('Registration Unsuccessful. Please contact support.');
         }
-      } else {
```

Comment:
if response.ok is false we want to know about it still - lets keep this else

Reply:
done. was actually added earlier missed that.